packages/api-docs-builder-core/muiSystem/getSystemComponentInfo.ts

```diff
@@ -0,0 +1,107 @@
+import fs from 'fs';
+import path from 'path';
+import kebabCase from 'lodash/kebabCase';
+import { getHeaders, getTitle } from '@mui/markdown';
+import {
+  ComponentInfo,
+  extractPackageFile,
+  getMuiName,
+  parseFile,
+  fixPathname,
+} from '@mui-internal/api-docs-builder/buildApiUtils';
+import findPagesMarkdown from '@mui-internal/api-docs-builder/utils/findPagesMarkdown';
+
+const migratedBaseComponents = [
+  'Badge',
+  'Button',
+  'ClickAwayListener',
+  'FocusTrap',
+  'Input',
+  'MenuItem',
+  'Menu',
+  'Modal',
+  'NoSsr',
+  'OptionGroup',
+  'Option',
+  'Popper',
+  'Portal',
+  'Select',
+  'Slider',
+  'Switch',
+  'TablePagination',
+  'TabPanel',
+  'TabsList',
+  'Tabs',
+  'Tab',
+];
```

Comment:
Could be imported from the dedicated file you created, or deleted if not usefull for system component